sensor_stack/trax/src/trax.py

```diff
@@ -30,7 +30,7 @@ def getInfo():
     imu.write(modInfo)
```

Comment:
Legacy file. You must revert changes done in this file. We are not using it anymore.